modules/bvl_feedback/ajax/new_bvl_feedback.php

```diff
@@ -33,7 +38,10 @@
         $_POST['comment'],
         'Y'
     );
+    //build url for email
+    $link = $feedbackThread->getFeedbackURL();
 
+    $newNotifier->notify(array("feedback" => $link));
```

Comment:
minor thing but this looks like it could probably be refactored so that you don't have to put this method invocation inside every `if` block